src/App.css

```diff
@@ -1,38 +1,109 @@
-.App {
-  text-align: center;
+/* Import */
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+/* Root */
+:root {
+    --primary-color: #ff6000;
+    --primary-dark: #282b2d;
+    --primary-grey: #93979c;
+    --light-grey: #c9d6df;
+    --white: #ffffff;
+    --red: #ff005c;
+    --green: #2eb872;
+    --yellow: #ffc300;
+    --blue: #0074e4;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+/* App */
+.app {
+    font-family: Inter, sans-serif;
+    margin: 0;
+    padding: 0;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+/* Buttons */
+.btn {
+    padding: 0.5rem 1rem;
+    border-radius: 1rem;
+    border: 3px solid var(--yellow);
+    cursor: pointer;
+    margin: 0.5rem 0;
+}
+.btn:focus{
+    outline: none;
```

Comment:
I think we shouldn't use outline:none !

Reply:
Thanks for making that point. I'll do it.